src/main/resources/com/cloudbees/jenkins/support/filter/DefaultContentFilter/index.jelly

```diff
@@ -0,0 +1,60 @@
+<?jelly escape-by-default='true'?>
+<!--
+  ~ The MIT License
+  ~
+  ~ Copyright (c) 2018, CloudBees, Inc.
+  ~
+  ~ Permission is hereby granted, free of charge, to any person obtaining a copy
+  ~ of this software and associated documentation files (the "Software"), to deal
+  ~ in the Software without restriction, including without limitation the rights
+  ~ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+  ~ copies of the Software, and to permit persons to whom the Software is
+  ~ furnished to do so, subject to the following conditions:
+  ~
+  ~ The above copyright notice and this permission notice shall be included in
+  ~ all copies or substantial portions of the Software.
+  ~
+  ~ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  ~ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  ~ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+  ~ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  ~ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  ~ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+  ~ THE SOFTWARE.
+  -->
+
+<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" xmlns:f="/lib/form" xmlns:st="jelly:stapler">
+    <l:layout title="${it.displayName}" permission="${it.CREATE_BUNDLE}">
+        <st:include page="sidepanel.jelly" it="${app}"/>
+        <l:main-panel>
+            <h1>Anonymized Items</h1>
+            <f:form action="update" method="post" name="Update Settings">
+                <f:entry field="enabled" title="Enabled" description="Whether or not to enable anonymization of support bundles">
+                    <f:checkbox checked="${it.enabled}"/>
+                </f:entry>
+                <f:block>
+                    <f:submit value="Update Settings"/>
+                </f:block>
+            </f:form>
+            <h2>Mappings</h2>
+            <table class="pane sortable bigtable">
+                <tr>
+                    <th class="pane-header">Actual Name</th>
+                    <th class="pane-header" initialSortDir="up">Anonymized Name</th>
+                </tr>
+                <j:forEach items="${it.mappings}" var="item">
+                    <tr>
+                        <td class="pane"><st:out value="${item.key}"/></td>
+                        <td class="pane"><st:out value="${item.value}"/></td>
```

Comment:
FYI the `<st:out` tag is not required here as jelly is escaping the variables (due to the first line)

But take care, OTOH the `<j:out` cannot be used here because in such case you open the way for injection. That's a bit confusion between `st:out` and `j:out` and just wanted to highlight that point.

Reply:
Concerning that, in general the escape tag is placed after the license, to keep all the code "close", to avoid such situation where you forgot that you have that line.